src/NzbDrone.Api/Queue/QueueResource.cs

```diff
@@ -17,7 +19,10 @@ public class QueueResource : RestResource
         public decimal Size { get; set; }
         public string Title { get; set; }
         public decimal Sizeleft { get; set; }
+
+        [JsonConverter(typeof(TimeSpanConverter))]
         public TimeSpan? Timeleft { get; set; }
```

Comment:
Yeah we can, I wanted to look at the usages first, in some cases having milliseconds might be useful (run time of a task perhaps).